gen_deb.sh

```diff
@@ -0,0 +1,31 @@
+#! /bin/bash
+
+export PACKAGE_NAME="libaedile-dev"
+export VERSION="0.0.2"
+
+export SRC_ROOT_DIR=$(dirname $(readlink -f ${0}))
+export DEB_PACKAGE_ROOT="${SRC_ROOT_DIR}/deb"
+export INSTALL_DIR="${DEB_PACKAGE_ROOT}/opt/${PACKAGE_NAME}/usr"
+
+
+rm -rf $INSTALL_DIR
+mkdir -p $INSTALL_DIR
+
+cd "${SRC_ROOT_DIR}"
+cmake --preset=linux
+cmake --build build/linux
+cmake --install build/linux
+
+rm -rf ${DEB_PACKAGE_ROOT}/usr/
+
+mkdir -p ${DEB_PACKAGE_ROOT}/usr/include/aedile/client
+mkdir -p ${DEB_PACKAGE_ROOT}/usr/lib
+
+ln -s ${INSTALL_DIR}/include/nostr.h ${DEB_PACKAGE_ROOT}/usr/include/aedile/nostr.h
```

Comment:
Why are we adding this `nostr.h` file?

Reply:
I think we will eventually have just a `nostr.hpp` that, in turn, imports everything from the project internals.  But we're not there yet.